apps/drec-api/src/utils/types.ts

```diff
@@ -0,0 +1,12 @@
+import { ICertificate as IOriginalCertificate } from '@energyweb/origin-247-certificate';
+
+/**
+ * Certificate source allows to describe how the certificate was issued.
+ * It can be issued by various entities like battery or generator itself
+ */
+export interface ICertificateMetadata {
+  deviceIds: number[];
+  id: string;
```

Comment:
I'm not picky, but can we change this to `groupId`? 

By the way if some devices are issued without the group, then this should be `null`. Any other value (like `0`) will be "magic", and not explicit, and will cause a problem sooner or later (somebody will query by that or something)

I think if we have `groupId: null | string` it will be as explicit as possible

Reply:
Sure